builds/e2e/connectivity.yaml

```diff
@@ -98,7 +98,10 @@ jobs:
             echo "deploymentFileName=$(deploymentFileName)"
             echo "test.runtimeLogLevel=$(test.runtimeLogLevel)"
       - template: templates/set-run-flag.yaml
-      - checkout: none
+      - checkout: self
+        clean: true
+        fetchDepth: 100
+        submodules: recursive
```

Comment:
We should make sure to match the pattern in other pipelines. Might be a good idea to create a template in the future.

Reply:
I talked with Damon about this before. It's matched with SN LH pipeline right now.